examples/minimq-stm32h7/stm32h7-ethernet/Cargo.toml

```diff
@@ -20,7 +20,10 @@ cortex-m = "0.6.2"
 log = "0.4.8"
 
 [dependencies.smoltcp]
-version = "0.6.0"
```

Comment:
Honestly, I'd prefer if we just bumped the hal version of stm32h7xx-hal to the most recent and used ethernet from there - they recently got merged together.